src/pages/Users.tsx

```
@@ -0,0 +1,556 @@
+    });
+    setIsEditDialogOpen(true);
+  };
+
+  const handleUpdateUser = async (e: React.FormEvent) => {
+    e.preventDefault();
+    if (!currentUser) return;
+
+    // Only include password if it's provided
+    const updateData = { ...formData };
+    if (!updateData.password) {
+      delete updateData.password;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)